make deploy failed

```
$ bikeshed update --skip-manifest
Downloading anchor data...
Downloading data for spec 1/217...
...
$ make local
bikeshed spec fetch.bs fetch.html --md-Text-Macro="COMMIT-SHA LOCAL COPY"
LINK ERROR: No 'dfn' refs found for 'creating' with for='['ReadableStream']'.
<a data-link-for="ReadableStream" data-link-type="dfn" data-lt="creating">creating</a>
...
$ make deploy
...
LINK ERROR: No 'dfn' refs found for 'creating' with for='['ReadableStream']'.
&lt;a data-link-for="ReadableStream" data-link-type="dfn" data-lt="creating">creating&lt;/a>
 ✘  Did not generate, due to fatal errorsmake: *** [Makefile:23: deploy] Error 22
```
This is also happening on PR check: https://github.com/whatwg/fetch/pull/1204/checks

--- a/fetch.bs
+++ b/fetch.bs
@@ -5181,12 +5181,13 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
  <li><p>Let <var>sizeAlgorithm</var> be an algorithm that accepts a <a>chunk</a> object and returns
  a non-negative, non-NaN, non-infinite number, chosen by the user agent.
 
- <li><p>Let <var>stream</var> be the result of <a for=ReadableStream>creating</a> a
- {{ReadableStream}} with <a for=ReadableStream/create><var>pullAlgorithm</var></a> set to
- <var>pullAlgorithm</var>, <a for=ReadableStream/create><var>cancelAlgorithm</var></a> set to
- <var>cancelAlgorithm</var>, <a for=ReadableStream/create><var>highWaterMark</var></a> set to
- <var>highWaterMark</var>, and <a for=ReadableStream/create><var>sizeAlgorithm</var></a> set to
- <var>sizeAlgorithm</var>.
+ <li><p>Let <var>stream</var> be a <a>new</a> {{ReadableStream}}.
+
+ <li><p><a for=ReadableStream>Set up</a> <var>stream</var> with
+ <a for="ReadableStream/set up"><var>pullAlgorithm</var></a> set to <var>pullAlgorithm</var>,
+ <a for="ReadableStream/set up"><var>cancelAlgorithm</var></a> set to <var>cancelAlgorithm</var>,
+ <a for="ReadableStream/set up"><var>highWaterMark</var></a> set to <var>highWaterMark</var>, and
+ <a for="ReadableStream/set up"><var>sizeAlgorithm</var></a> set to <var>sizeAlgorithm</var>.
 
  <li>
   <p>Run these steps, but <a>abort when</a> the ongoing fetch is <a for=fetch>terminated</a>:
@@ -5986,7 +5987,8 @@ steps:
 
 <ol>
  <li><p>Let <var>stream</var> be <var>object</var> if <var>object</var> is a {{ReadableStream}}
- object; otherwise the result of <a for=ReadableStream>creating</a> a {{ReadableStream}}.
+ object. Otherwise, let <var>stream</var> be a <a>new</a> {{ReadableStream}}, and
+ <a for=ReadableStream>set up</a> <var>stream</var>.
 
  <li><p>Let <var>action</var> be null.
 

